### DISCUSSION TITLE
Is there a way to disable CDN in code?

### DISCUSSION THREAD

**Discussion by HengzheLi** [OrchardCMS/OrchardCore#6986] [Q&A] [answered]
Looks like the PR is under review https://github.com/OrchardCMS/OrchardCore/pull/5969. As I create a new tenant with default CDN settings for now, it will use the CDN to load the resources Jquery-UI etc. But the loading is failed and I try to disable the CDN with below code when the application starts up
`
services.Configure<ResourceManagementOptions>(options =>
            {
                options.UseCdn = false;
            });
`
But it does not work.

Has a way to achive that?

**Comment by ns8482e** [comment, marked as answer]
There is Site setting that let you configure whether to use CDN or local scripts

**Reply by congzw** [reply]
but how?
